INSTALL

```diff
@@ -18,6 +18,7 @@ Runtime dependencies:
 
     - bash (if you use the default autostart file)
     - libx11
+    - xft and freetype
```

Comment:
Should we add the new requirements to NEWS, too? It is kind of "user visible" and will probably make the life of package maintainers easier

Reply:
Yes, good idea! I've added them.

Reply:
Thanks!